app/uk/gov/hmrc/eusubsidycompliancestub/models/SubsidyUpdate.scala

```diff
@@ -68,7 +68,7 @@ object SubsidyUpdate {
         } else
           UndertakingSubsidyAmendment(
             (json \ "undertakingSubsidyAmendment").as[List[NonHmrcSubsidy]]
-          ) // TODO this can only be this if we can get the API aligned
```

Comment:
It's a TODO from a few years ago. What it means is not clear in the current state of the project.